src/realm/db.cpp

```diff
@@ -2931,27 +2931,9 @@ DisableReplication::~DisableReplication()
         m_tr.initialize_replication();
 }
 
-StringInterner* DB::get_string_interner(TableKey tk, ColKey::Idx col_idx)
+StringInterner* DB::get_string_interner(TableKey tk, ColKey col_key)
```

Comment:
Do we still need this? We are not expecting to reach this function, don't we?

Reply:
You're right, I'll remove it in the target branch.